hockey-score-front-end/src/app/players/add-player/add-player.component.html

```diff
@@ -0,0 +1,95 @@
+                    </div>
+                </div>
+                <div class="form-group col">
+                    <label>Weight</label>
+                    <input type="text" formControlName="weight" class="form-control"
+                        [ngClass]="{ 'is-invalid': submitted && f.weight.errors }" />
+                    <div *ngIf="submitted && f.weight.errors" class="invalid-feedback">
+                        <div *ngIf="f.weight.errors.required">Weight is required</div>
+                    </div>
+                </div>
+                <div class="form-group col">
+                    <label>Height</label>
+                    <input type="text" formControlName="height" class="form-control"
+                        [ngClass]="{ 'is-invalid': submitted && f.height.errors }" />
+                    <div *ngIf="submitted && f.height.errors" class="invalid-feedback">
+                        <div *ngIf="f.height.errors.required">Height is required</div>
+                    </div>
+                </div>
+                <div class="form-group col">
+                    <label>Age</label>
+                    <input type="text" formControlName="age" class="form-control"
+                        [ngClass]="{ 'is-invalid': submitted && f.age.errors }" />
+                    <div *ngIf="submitted && f.age.errors" class="invalid-feedback">
+                        <div *ngIf="f.age.errors.required">Age is required</div>
+                    </div>
+                </div>
+
+            </div>
+
+            <div class="form-group col">
+                <label>Shoots</label>
+                <select formControlName="shoots" class="form-control"
+                    [ngClass]="{ 'is-invalid': submitted && f.shoots.errors }">
+                    <option value=""></option>
+                    <option value="left">Left</option>
+                    <option value="right">Right</option>
+                </select>
+                <div *ngIf="submitted && f.shoots.errors" class="invalid-feedback">
+                    <div *ngIf="f.shoots.errors.required">Shoots is required</div>
+                </div>
+            </div>
+
+            <div class="form-group col">
+                <label>Country</label>
+                <select formControlName="country" class="form-control"
+                    [ngClass]="{ 'is-invalid': submitted && f.country.errors }">
+                    <option value=""></option>
+                    <option value="by">Belarus</option>
+                    <option value="ru">Russian</option>
+                </select>
+                <div *ngIf="submitted && f.country.errors" class="invalid-feedback">
+                    <div *ngIf="f.country.errors.required">Country is required</div>
+                </div>
+            </div>
+
+            <div class="text-center">
+                <button class="btn btn-primary mr-1">Register</button>
```

Comment:
As it said in the task - we should disable button if there are some validation errors in the form or if it's empty.

Reply:
By the way, you have already used this approach on Login Page form